.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+    "singleQuote": true,
+    "trailingComma": "all",
```

Comment:
this is totally opinionated, same as arrow parenthesis for single arg, why are you adding them?

Reply:
It's the general configuration used in many projects.
what's your opinion? What should be done?

Reply:
Without, please keep in mind current style.